9.4.Методы_строк_Часть_2/9.4.12.py

```diff
@@ -0,0 +1,6 @@
+s = input()
+
+if s.endswith(".com") == True or s.endswith(".ru") == True:
```

Comment:
1. Не нужно проверять на истинность, потому что метод `.endswith()` сам по себе отдает булевое значение.
2. Чтобы не строить длинное логическое выражение, можно передать кортеж значений в аргумент методу `.endswith()`:
```suggestion
if s.endswith((".com", ".ru")):
```